commands/web-searches/duck-duck-go.sh

```diff
@@ -8,6 +8,6 @@
 
 # Optional parameters:
 # @raycast.icon images/duck-duck-go.png
-# @raycast.argument1 { "type": "text", "placeholder": "query" }
+# @raycast.argument1 { "type": "text", "placeholder": "query","percentEncoded": true}
 
 open "https://duckduckgo.com/?q=${1// /%20}"
```

Comment:
If you are going to add the `percentEncoded` to the argument, you will need to change the `open` instruction to:
```suggestion
open "https://duckduckgo.com/?q=$1"
```